ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.cpp

```diff
@@ -202,7 +202,7 @@ uint32_t get_num_cores_channels_from_parallel_config(const ParallelConfig& pconf
 }
 
 MemoryConfig create_sharded_memory_config_from_parallel_config(
-    const ttnn::Shape& tensor_shape, ParallelConfig& parallel_config, uint32_t tile_size) {
+    const ttnn::Shape& tensor_shape, ParallelConfig& parallel_config, uint32_t tile_size, bool is_tiled) {
```

Comment:
why need another bool? Can we not infer from the `tile_size` arg?

Reply:
https://github.com/tenstorrent/tt-metal/pull/13291#discussion_r1800584198
It was to address this comment.

Reply:
removed it.